apps/vr-tests-react-components/src/stories/Field.stories.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+import Screener, { Steps } from 'screener-storybook/src/screener';
+import { storiesOf } from '@storybook/react';
+import { Radio } from '@fluentui/react-radio';
+import {
+  CheckboxField,
+  ComboboxField,
+  InputField,
+  InputFieldProps,
+  RadioGroupField,
+  SelectField,
+  SliderField,
+  SpinButtonField,
+  SwitchField,
+  TextareaField,
+} from '@fluentui/react-field';
+import { SparkleFilled } from '@fluentui/react-icons';
+
+const AllFields = (
+  props: Pick<
+    InputFieldProps,
+    'orientation' | 'required' | 'label' | 'validationState' | 'validationMessage' | 'validationMessageIcon' | 'hint'
+  >,
+) => {
+  return (
+    <div style={{ display: 'grid', rowGap: '12px' }}>
+      <CheckboxField label="Checkbox" {...props} />
+      <ComboboxField label="Combo box field" {...props} />
+      <InputField label="Input field" {...props} />
+      <RadioGroupField label="Radio group field" {...props}>
+        <Radio label="Option one" />
+        <Radio label="Option two" />
+        <Radio label="Option three" />
+      </RadioGroupField>
+      <SelectField label="Select field" {...props}>
+        <option>Option</option>
+      </SelectField>
+      <SliderField label="Slider field" {...props} />
+      <SpinButtonField label="Spin button field" {...props} />
+      <SwitchField label="Switch field" {...props} />
+      <TextareaField label="Textarea field" {...props} />
+    </div>
+  );
+};
+
+const AllFieldsWithValidationMessage = (
+  props: Pick<InputFieldProps, 'orientation' | 'required' | 'label' | 'hint'>,
+) => {
+  type ValidationProps = Pick<InputFieldProps, 'validationState' | 'validationMessage' | 'validationMessageIcon'>;
+  const error: ValidationProps = { validationState: 'error', validationMessage: 'Error message' };
+  const warning: ValidationProps = { validationState: 'warning', validationMessage: 'Warning message' };
+  const success: ValidationProps = { validationState: 'success', validationMessage: 'Success message' };
+  const custom: ValidationProps = { validationMessage: 'Custom message', validationMessageIcon: <SparkleFilled /> };
+
+  return (
+    <div style={{ display: 'grid', rowGap: '12px' }}>
+      <CheckboxField label="Checkbox" {...error} {...props} />
+      <ComboboxField label="Combo box field" {...warning} {...props} />
+      <InputField label="Input field" {...success} {...props} />
```

Comment:
Do you think it's worth having a snapshot of each validation state for each component? IIRC there is some design discussion around having `InputField` have a full red border in its error state which would not be captured with this test as-is.

Or perhaps an individual story like "Checkbox+fieldLabel" if/when that visual state is implemented?

Reply:
That's a good point. I initially had it that way, but was just trying to reduce the number of snapshots. I'll switch it back to having a separate snapshot of each component with each validation state, so we can capture the differences in how the components render in each state.